File: QolmsYappli/QolmsYappli/Models/InnerModel/DatachargeHistItem.vb
''' <summary>
''' データチャージの履歴を表します。
''' このクラスは継承できません。
''' </summary>
''' <remarks></remarks>
<Serializable()>
Public NotInheritable Class DatachargeHistItem

#Region "Public Property"

    ''' <summary>
    ''' 操作日時を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property ActionDate As Date = Date.MinValue

    ''' <summary>
    ''' データサイズを取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property Size As Integer = Integer.MinValue

    ''' <summary>
    ''' ポイントを取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property Point As Integer = Integer.MinValue

    ''' <summary>
    ''' 表示名を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property DispName As String = String.Empty

#End Region

#Region "Constructor"

    ''' <summary>
    ''' <see cref="DatachargeHistItem" /> クラスの新しいインスタンスを初期化します。
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()
    End Sub

#End Region

End Class

File: QolmsJotoWebView/Models/InnerModels/DatachargeHistItem.cs
using System;

namespace MGF.QOLMS.QolmsJotoWebView
{
    /// <summary>
    /// データチャージの履歴を表します。
    /// このクラスは継承できません。
    /// </summary>
    [Serializable()]
    public sealed class DatachargeHistItem
    {
        #region Public Property

        /// <summary>
        /// 操作日時を取得または設定します。
        /// </summary>
        public DateTime ActionDate { get; set; } = DateTime.MinValue;

        /// <summary>
        /// データサイズを取得または設定します。
        /// </summary>
        public int Size { get; set; } = int.MinValue;

        /// <summary>
        /// ポイントを取得または設定します。
        /// </summary>
        public int Point { get; set; } = int.MinValue;

        /// <summary>
        /// 表示名を取得または設定します。
        /// </summary>
        public string DispName { get; set; } = string.Empty;

        #endregion

        #region Constructor

        /// <summary>
        /// <see cref="DatachargeHistItem" /> クラスの新しいインスタンスを初期化します。
        /// </summary>
        public DatachargeHistItem()
        {
        }

        #endregion
    }
}